gomock_generator/gomockgenerator/generate.go

```diff
@@ -67,7 +69,12 @@ func GenerateMocks(ctx context.Context, gcfg GenerationConfiguration, mocksCfg M
 	if mocksCfg.BaseDirectory == "" {
 		mocksCfg.BaseDirectory = mocksCfg.BasePackage
 	}
-	err = os.Chdir(path.Join(os.Getenv("GOPATH"), "src", mocksCfg.BaseDirectory))
+
+	mocksPath := path.Join(path.Join(os.Getenv("GOPATH"), "src"), mocksCfg.BaseDirectory)
```

Comment:
issue: `path.Join` receives variadic arguments
```suggestion
	mocksPath := path.Join(os.Getenv("GOPATH"), "src", mocksCfg.BaseDirectory)
```

Reply:
Yep. indeed.